src/Equinox.EventStore/EventStore.fs

```diff
@@ -685,4 +747,4 @@ type Connector
             let! masterInParallel = Async.StartChild (__.Connect(name + "-TwinW", discovery, NodePreference.Master))
             let! slave = __.Connect(name + "-TwinR", discovery, NodePreference.PreferSlave)
             let! master = masterInParallel
-            return Connection(readConnection=slave, writeConnection=master, ?readRetryPolicy=readRetryPolicy, ?writeRetryPolicy=writeRetryPolicy) }
\ No newline at end of file
+            return Connection(readConnection = slave, writeConnection = master, ?readRetryPolicy = readRetryPolicy, ?writeRetryPolicy = writeRetryPolicy) }
```

Comment:
wow 62 lines?! the Cosmos count is going to go insane!

Reply:
Also, Cosmos.fs has 2 PRs that need rebasing and doing this would make that more painful so prob best to leave that, I reckon

Reply:
Yes, I was planning to only do ES for now :)